### PR TITLE
Interface round1

### LINKED ISSUE #8: Audio

### PR DESCRIPTION
**Pull request by nsakhnini**
closes #8 
Audio and resources:
http://soundbible.com/147-UFO-Landing.html
http://soundbible.com/1458-Monster-Roar.html
https://freesound.org/people/NenadSimic/sounds/171697/
https://freesound.org/people/NenadSimic/sounds/171756/
https://freesound.org/people/NenadSimic/sounds/268108/
https://freesound.org/people/NenadSimic/sounds/149966/
https://3dwarehouse.sketchup.com/model/e015eee7-7b2c-49a1-8dee-6923c9945a88/Alien-Queen

### CODE REVIEW

File: Andy Solar System Test/Assets/rotate.cs [hunk 1]
```diff
@@ -5,16 +5,20 @@
...
-		
+
 	}
-	
+
 	// Update is called once per frame
 	void Update () {
 
-		transform.RotateAround (Vector3.up, rotateSpeed * Time.deltaTime);
+		transform.Rotate (Vector3.up, rotateSpeed* revolutionSpeed * Time.deltaTime);
```
Comment (jhopp3) [inline Andy Solar System Test/Assets/rotate.cs]: This is probably right, but I don't understand the change.  Can you explain please?

File: Andy Solar System Test/Assets/Planets.cs [hunk 2]
```diff
@@ -10,32 +10,52 @@
...
+	[SerializeField] public float rotationSpeedChange = 1.0F;
+	[SerializeField] public float timePassingChange ;
+
+	[SerializeField] private List<GameObject> allPlanetsCenters = new List<GameObject>() ;
+
 	float orbitWidth = 0.01F;
 	float habWidth = 0.03F;
 
-	float revolutionSpeed = 0.2F;
+	float revolutionSpeed ;
```
[the remark this answers is not in the document]
Comment (nsakhnini) [reply Andy Solar System Test/Assets/Planets.cs]: I set that using the slide in the menu.

File: Andy Solar System Test/Assets/JsonHelper.cs [hunk 3]
```diff
@@ -5,29 +5,29 @@
 
 public static class JsonHelper
 {
-    public static T[] FromJson<T>(string json)
-    {
-        Wrapper<T> wrapper = JsonUtility.FromJson<Wrapper<T>>(json);
-        return wrapper.Items;
-    }
+	public static T[] FromJson<T>(string json)
```
[the remark this answers is not in the document]
Comment (nsakhnini) [reply Andy Solar System Test/Assets/JsonHelper.cs]: Nop!

File: Andy Solar System Test/Assets/rotate.cs (hunk 1 above)
Comment (nsakhnini) [reply Andy Solar System Test/Assets/rotate.cs]: Rotate around is deprecated. It didn't work correctly when I changed it.

File: Andy Solar System Test/Assets/Planets.cs [hunk 4]
```diff
@@ -74,7 +97,7 @@ public class Planets : MonoBehaviour {
 
 			float planetDistance = float.Parse (planets [planetCounter, 0]) / 149600000.0F * 10.0F;
 			float planetSize = float.Parse (planets [planetCounter, 1]) * 2.0F / 10000.0F;
-			float planetSpeed = -1.0F / float.Parse (planets [planetCounter, 2]) * revolutionSpeed;
+			float planetSpeed = -1.0F / float.Parse (planets [planetCounter, 2]);
```
[the remark this answers is not in the document]
Comment (nsakhnini) [reply Andy Solar System Test/Assets/Planets.cs]: Just did that in the rotate class. I thought no need to change the speed for each planet alone.

File: Andy Solar System Test/Assets/JsonHelper.cs (hunk 3 above)
[the remark this answers is not in the document]
Comment (nsakhnini) [reply Andy Solar System Test/Assets/JsonHelper.cs]: I didn't change it

**Pull request merged**